autovalet_system/src/autovalet_system.py

```diff
@@ -41,7 +44,6 @@ class State:
 class AutoValet:
     def __init__(self,sim):
 
-
```

Comment:
rip this whitespace line. we will miss it

Reply:
:reminder_ribbon: